_data/banking.yml

```diff
@@ -521,14 +521,22 @@ websites:
     twitter: Holvi
     facebook: HolviCom
 
-  - name: HSBC
-    url: https://www.hsbc.com/
+  - name: HSBC UK
+    url: https://www.hsbc.co.uk/
     img: hsbc.png
     tfa:
       - proprietary
       - hardware
     doc: https://www.hsbc.co.uk/help/security-centre/secure-key/secure-key-help/
 
+  - name: HSBC US
+    url: https://www.us.hsbc.com/
+    img: hsbc.png
+    twitter: HSBC_US
+    facebook: HSBCUS
+    status: https://www.us.hsbc.com/online-banking/security-device/
```

Comment:
The status tag is designed to link users to a blog post or tweet from the company explaining when 2FA will be rolled out.
```suggestion
```

Reply:
Do we have a field for this specific purpose, ie explaining why a company that seemingly implemented 2FA didn't? (And still shame them)